app/src/main/java/org/pesmypetcare/mypetcare/activities/views/chart/statisticdata/StatisticData.java

```diff
@@ -73,17 +74,21 @@ public double getyMaxValue() {
      * @return The date that has been read
      */
     protected String getDate(DateTime dateTime) {
-        int year = dateTime.getYear();
         int month = dateTime.getMonth();
         int day = dateTime.getDay();
-        StringBuilder date = new StringBuilder(String.valueOf(year));
-        date.append('-');
+        StringBuilder date = new StringBuilder("");
+
+        if (day < FIRST_TWO_DIGITS_DAY) {
+            date.append('0');
+        }
+
+        date.append(day).append("-");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid appending characters as strings in StringBuffer.append.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5593371)